Does not properly compile on Windows

I have not tried to test this on windows yet. I do have access to a windows machine, but no idea how we can get this automated.

First "quick" try, by the way I'm no pro in compiling this things.
Probably something with my "Microsoft C++ Build Tools" install is still wrong...

```
==> exqlite

Microsoft (R) Program Maintenance Utility Version 14.28.29913.0
Copyright (C) Microsoft Corporation.  All rights reserved.

        del /Q /F priv
        erl -noshell -s init stop -eval "io:setopts(standard_io, [{encoding, unicode}]), io:format(\"ERTS_INCLUDE_PATH=~ts/erts-~ts/include/\", [code:root_dir(), erlang:system_info(version)])." > Makefile.auto.win
        nmake /                   /F Makefile.win priv\sqlite3_nif.dll

Microsoft (R) Program Maintenance Utility Version 14.28.29913.0
Copyright (C) Microsoft Corporation.  All rights reserved.

        if NOT EXIST "priv" mkdir "priv"
        cl /O2 /EHsc /I"sqlite3" /I"c:/Users/cw/scoop/apps/erlang/23.2.7/erts-11.1.8/include/" /LD /MD /Fepriv\sqlite3_nif.dll sqlite3\sqlite3.c c_src\sqlite3_nif.c c_src\queue.c
Microsoft (R) C/C++ Optimizing Compiler Version 19.28.29913 for x64
Copyright (C) Microsoft Corporation.  All rights reserved.

sqlite3.c
sqlite3\sqlite3.c(1098): fatal error C1083: Cannot open include file: 'stdarg.h': No such file or directory
sqlite3_nif.c
c_src\sqlite3_nif.c(1): fatal error C1083: Cannot open include file: 'assert.h': No such file or directory
queue.c
c1: fatal error C1083: Cannot open source file: 'c_src\queue.c': No such file or directory
Generating Code...
NMAKE : fatal error U1077: '"C:\Program Files (x86)\Microsoft Visual Studio\2019\BuildTools\VC\Tools\MSVC\14.28.29910\bin\Hostx64\x64\cl.EXE"' : return code '0x2'
Stop.
NMAKE : fatal error U1077: '"C:\Program Files (x86)\Microsoft Visual Studio\2019\BuildTools\VC\Tools\MSVC\14.28.29910\bin\Hostx64\x64\nmake.EXE"' : return code '0x2'
Stop.
```

This line at least is a bug in `Makefile.win`:
```
c1: fatal error C1083: Cannot open source file: 'c_src\queue.c': No such file or directory
Generating Code...
```

The file specifies a `queue.c` that indeed does not exist in the code! Likely an artifact of one of the older sqlite NIFs we adapted. Removing that should fix that issue. 

The rest I have no idea. Looking at the `stdarg.h` thing, it seems like that may indicate some sort of misconfigured build tool, see this issue I found:

https://github.com/microsoft/vscode/issues/75334

--- a/Makefile.win
+++ b/Makefile.win
@@ -9,7 +9,7 @@
 NMAKE = nmake /$(MAKEFLAGS)
 CFLAGS = /O2 /EHsc /I"sqlite3"
 
-SRC = sqlite3\sqlite3.c c_src\sqlite3_nif.c c_src\queue.c
+SRC = sqlite3\sqlite3.c c_src\sqlite3_nif.c
 
 all: clean priv\sqlite3_nif.dll
 

--- a/guides/windows.md
+++ b/guides/windows.md
@@ -1,0 +1,44 @@
+# Windows Users
+
+> Exqlite uses an [Erlang NIF](https://erlang.org/doc/tutorial/nif.html) under the hood.  
+> Means calling a native implementation in C.  
+> For Windows users this means compiling Exqlite does not magically just work.  
+
+## Requirements
+
+### Install Microsoft Visual C++ Build Tools
+
+Download page:  
+[visualstudio.microsoft.com/visual-cpp-build-tools](https://visualstudio.microsoft.com/visual-cpp-build-tools/)
+
+Alternative direct download link:  
+[aka.ms/vs/16/release/vs_buildtools.exe](https://aka.ms/vs/16/release/vs_buildtools.exe)  
+_for Visual Studio 2019 - version 16_
+
+## Building
+
+### Start command prompt with necessary environment
+
+> Assuming you want to build for Windows x64.
+
+Within Windows Start menu search for:  
+x64 Native Tools Command Prompt
+
+Starting this command prompt all necessary environment variables for compiling should be ready.
+
+Means ready to run:
+```cmd
+mix deps.compile exqlite
+
+# or
+mix compile
+mix test
+```
+
+**Alternative way to start prompt**
+
+Assuming you have _latest_ version of Build Tools (Visual Studio 2019) installed.
+
+```
+cmd /k "C:\Program Files (x86)\Microsoft Visual Studio\2019\BuildTools\VC\Auxiliary\Build\vcvars64.bat"
+```

